storage/UsedFeatures.h

```diff
@@ -61,6 +61,7 @@ namespace storage
 	UF_FCOE = 1 << 18,
 	UF_FC = 1 << 19,
 	UF_DASD = 1 << 20,
+	UF_PMEM = 1 << 29,
```

Comment:
Hmm, maybe nitpicking, but it seems that this should be sorted somehow better. Extending this more and more can cause errors.

Reply:
It is sorted into categories.